templates/zerver/integrations/email.md

```diff
@@ -1,8 +1,9 @@
-You can send emails to Zulip! This is useful:
+You can send emails to Zulip!
 
-* If you have an email that you want to discuss on Zulip
-* For mirroring mailing list traffic
-* For integrating a third-party app that can send emails, but which does not
-  easily lend itself to more direct integration
+1.  Instructions are available [here](/help/message-a-stream-by-email).
+    This is useful:
 
-Instructions are available [here](/help/message-a-stream-by-email).
+    * If you have an email that you want to discuss on Zulip
+    * For mirroring mailing list traffic
+    * For integrating a third-party app that can send emails, but which does not
+      easily lend itself to more direct integration
```

Comment:
I'm not sure that for items like the "Email" one, it looks better to have steps?

Maybe the right model is to just tweak the text a bit -- move the bulleted list of reasons this might be useful to the top as sorta the opening advertisement, and then have the bulleted list be just "1. Instructions are available [here](...).`